android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/PhysicalWebBroadcastService.java

```diff
@@ -62,7 +62,6 @@
     private NotificationManagerCompat mNotificationManager;
```

Comment:
URL should be all caps in commit message.  Subject message suggestion: Send URL to PWS before broadcasting